src/cmd/troubleshoot/context.go

```diff
@@ -14,8 +14,8 @@ type troubleshootContext struct {
 	context            context.Context
 	apiReader          client.Reader
 	httpClient         *http.Client
-	namespaceName      string // the default namespace ("dynatrace") or provided in the command line
-	dynakubeName       string // the default name of dynakube ("dynakube") or provided in the command line
+	namespaceName      string // the default namespace ("dynatrace") or value provided in the command line
+	dynakubeName       string // all dynakubes in namespaceName or value provided in the command line
```

Comment:
```suggestion
```
if the dynakube flag is passed as a parameter, I think this field is not needed anymore. I would avoid having a dynakube instance and a dynakube name both in a struct when the dynakube instance contains the same information.

Thinking about it, the namespace field has the same problem, as the name of the namespace is also stored in the dynakube instance

Reply:
The dynakube name is still used by the `checkDynakube` test, to get the actual dynakube. - Will move the `checkDynakubeCrdExists` from the "per dynakube" - tests to the one time tests. => then we can remove the dynakubename from the struct, agree.